application/src/main/java/org/togetherjava/tjbot/features/github/GitHubCommand.java

```diff
@@ -122,10 +134,13 @@ private ToIntFunction<String> suggestionScorer(String title) {
     }
 
     private void updateCache() {
+        logger.debug("GitHub Autocomplete cache update started"); // log start
+
         autocompleteGHIssueCache = reference.getRepositories().stream().map(repo -> {
             try {
                 return repo.getIssues(GHIssueState.ALL);
             } catch (IOException ex) {
+                logger.error("Error fetching issues from repo {}", repo.getName(), ex);
```

Comment:
we dont need that extra log. we will get this logged from the wrapping catch that you added in line 78 already. So with your current code we would get this logged twice, which is confusing.

I understand that you want to add the extra info which repo it was and that is great. We can achieve that without logging twice by adding the message to the exception we re-throw:

```java
throw new UncheckedIOException("Error fetching issues from repo " + repo.getName(), ex);
```